src/components/project/project.rules.ts

```diff
@@ -702,6 +713,10 @@ export class ProjectRules {
               to: ProjectStep.Completed,
               type: TransitionType.Approve,
               label: 'Complete 🎉',
+              disabled,
+              disabledLabel: disabled
+                ? 'The project cannot be Completed if any engagements have a non-terminal status'
```

Comment:
I think it would be better to replace the "if wording" with "assertion wording" since we know it's true.
```suggestion
                ? 'The project cannot be completed since some engagements have a non-terminal status'
```